docs/GettingStartedDocs/Contributors/SimulatorGettingStarted.md

```diff
@@ -98,7 +98,7 @@ Errors while running CTest
 
 Some of the tests are skipped (Not Run) by design because the current simulator is not fully featured yet.
 
-A clean pass of the above unitests run is an indication that your Open Enclave setup was successful. You can start playing with those Open Enclave samples after following the instructions in the "Install" section below to configure samples for building,
+A clean pass of the above unit tests run is an indication that your Open Enclave setup was successful. You can start playing with those Open Enclave samples after following the instructions in the "Install" section below to configure samples for building,
```

Comment:
Same comment on "unit tests run" as in SGX1FLCGettingStarted.md.   Also can you break this into multiple lines in the source md?

Reply:
Fixed.